ldes-server-infra-postgres/postgres-liquibase/src/main/resources/db/changelog/3_0_1/bucketisation.sql

```diff
@@ -0,0 +1,12 @@
+create table public.bucketisation
```

Comment:
Would reccomend it to split creation off in the XML approach, looks a bit cleaner :)